src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -196,4 +198,22 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
                 By.cssSelector("input[id$='OptionText-" + qnNumber + "-" + responseNumber + "']"));
         waitForElementToBeClickable(element);
     }
+    
+    // ------------- For InstructorEditStudentFeedbackPage -------------
+    
+    public boolean clickModerationHintButton() {
+        click(By.id("moderationHintButton"));
+        // Check if links toggle properly.
+        WebElement moderationHint = browser.driver.findElement(By.id("moderationHint"));
+        return moderationHint.isDisplayed();
```

Comment:
Indeed the test case in `InstructorFeedbackResultsPageUiTest` can be improved. While it is good to reference the existing code to see how things are currently being done, there are some parts which still suffer from code quality issues 😮 . In the above case, the method is used to click on an element, and simply it should just be doing that.